### ISSUE TITLE
the result with SVM

### ISSUE THREAD

**Issue by longmao-yiran** [facebook/C3D#373]
Hi Du,
I read the paper you wrote, but one of the things I didn't quite clear.You are using softmax loss as the loss function training network, which means the network weights and features adapt to the softmax loss.But using the SVM classifier is even better.Do you know the reason for this phenomenon?

**Comment by dutran** [comment]
The best result is to fine-tune the whole network. in case we dont fine-tune, liner SVM on fc6 should work best. Since we have another stage to average-pool the fc6 features before linear SVM. It make linear SVM learns to optimiza accuracy based on video-level features, thus have better context than simply aggregate softmax predictions where each prediction is made independently. **I just guess**
